modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/EditObjectDefinitionMVCActionCommand.java

```diff
@@ -101,6 +101,10 @@ protected void doProcessAction(
 				return;
 			}
 
+			if (objectDefinition.isEnableComments()) {
+				enableComments = true;
+			}
```

Comment:
@marcelabc 

I believe it should handle this by usigin validation, it can be done [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java#L1314-L1331)

this way the user that is using the API will recieve a feedback,

what do you think?